.env.example

```diff
@@ -0,0 +1 @@
+NODE_PROVIDER_MATIC_RPC_URL=https://polygon-mainnet.g.alchemy.com/v2/gHj4hjcafR_LP6vG6Ru6s1MomfYeVdYC
```

Comment:
the example file shouldn't have live urls

Reply:
hehe, I see! we may have to wait for @aspiers to shed some light there. I think he set it up on the mono repo

Reply:
We don't run tests in forking mode in tokenizer's CI.

Reply:
ah, really? I thought when I open a PR against upstream from my forked repo it will run the test suite. that's not what's happening @kargakis?

Reply:
@pheuberger the test suite still runs but in the hardhat network, there is no need to use forking mode. Here forking mode is used to leverage the deployed contracts from the live network.

Reply:
There is actually a place where we use forking mode ([storage layout checks](https://github.com/ToucanProtocol/tokenizer/blob/248b4e4d34b7aa8e9a1fcdd207b0fc4e39484fa1/.github/workflows/hardhat-deploy.yml#L76-L86)) but they work without having to use an archival node.